src/typescript-service.ts

```diff
@@ -1245,7 +1245,9 @@ export class TypeScriptService {
 
         return this.projectManager.ensureOwnFiles(span)
             .concat(Observable.defer(() => {
-                const configuration = this.projectManager.getConfiguration(fileTextChanges[0].fileName)
+                // Configuration lookup uses Windows paths, FileTextChanges uses unix paths. Convert to backslashes.
+                const firstChangedFile = uri2path(path2uri(fileTextChanges[0].fileName))
```

Comment:
If the purpose of this is to convert to backslashes, why not simply do `filename.replace(/\//g, '\\')`? Converting to a URI and back seems like a hack

Reply:
I see that the intention is to only do it if it's a path with a drive letter, I'd still prefer to leave the URI functions out of this

Reply:
Yes it's not the cleanest looking solution :) I'll extract the needed logic from uri2path.

Reply:
Done.